### ISSUE TITLE
Pro le

### ISSUE THREAD

**Issue by marithpl** [konradb3/homeassistant_salus#12]
I've got problem like this - after update HA to 0.118.5

2020-12-09 20:49:42 ERROR (MainThread) [pyit600] Timeout while connecting to gateway:                                                                      
2020-12-09 20:49:42 ERROR (MainThread) [pyit600] Timeout while connecting to gateway:                                                                      
2020-12-09 20:49:46 WARNING (MainThread) [homeassistant.config_entries] Config entry for airly not ready yet. Retrying in 5 seconds                        
2020-12-09 20:49:46 ERROR (MainThread) [custom_components.homeassistant_salus.climate] Authentication error: check if you have specified gateway's TOKEN co
rrectly.                                                                                                                                                   
2020-12-09 20:49:46 ERROR (MainThread) [homeassistant] Error doing job: Unclosed client session                                                            
2020-12-09 20:49:46 ERROR (MainThread) [homeassistant] Error doing job: Unclosed connector                                                                 
2020-12-09 20:49:46 ERROR (MainThread) [custom_components.homeassistant_salus.binary_sensor] Authentication error: check if you have specified gateway's TO
KEN correctly.                                                                                                                                             
2020-12-09 20:49:46 ERROR (MainThread) [homeassistant] Error doing job: Unclosed client session                                                            
2020-12-09 20:49:46 ERROR (MainThread) [homeassistant] Error doing job: Unclosed connector

**Comment by jvitkauskas** [comment]
Can you try https://github.com/jvitkauskas/homeassistant_salus ?

**Comment by marithpl** [comment]
@jvitkauskas I have only replace a files?

**Comment by jvitkauskas** [comment]
Yes and then restart. Also I would delete pycache folder if there is one present.
